api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+    "\n",
+    "initial_best_measured_value = min(toy_y)\n",
+    "\n",
+    "print(f\"Best (lowest) value in initial training set: {initial_best_measured_value}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can plot the initial training set, and color it by the function value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot initial training set, colored by toy function values\n",
+    "\n",
+    "plt.scatter(np.array(toy_x)[:,0], np.array(toy_x)[:,1],\n",
+    "            c=toy_y, cmap=plt.cm.plasma)\n",
+    "plt.xlabel(\"X1\")\n",
+    "plt.ylabel(\"X2\")\n",
+    "plt.xlim(-5,5)\n",
+    "plt.ylim(-5,5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# Write a PIF JSON dataset file\n",
+    "write_dataset_from_func(toy_func, \"toy_initial_dataset.json\", toy_x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a dataset, upload to Citrination, return/print the ID\n",
+    "\n",
+    "dataset_id = upload_data_and_get_id(\n",
+    "    client,\n",
+    "    \"toy_initial_dataset\",\n",
+    "    \"toy_initial_dataset.json\",\n",
+    "    create_new_version=True,\n",
+    "#   given_dataset_id=1234\n",
+    ")\n",
+    "print(f\"Dataset created: {dataset_id}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make a data view on Citrination, return/print the ID\n",
+    "# view_id = 1234 # If view exists and you're re-running this notebook\n",
+    "\n",
+    "view_id = build_view_and_get_id(client, dataset_id, \"toy_view\", view_desc=\"toy test view\",\n",
+    "                      input_keys=[\"Property x1\", \"Property x2\"], output_keys=[\"Property y\"])\n",
+    "\n",
+    "print(f\"Data view created: {view_id}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running design\n",
+    "\n",
+    "Now we call a function that runs the SL process. In short, we run a loop where we do the following:\n",
+    "\n",
+    "1. Submit a design run (and poll status w/ wait times)\n",
+    "1. Get best candidates and record predicted results\n",
+    "1. Measure candidates (using toy function evaluation) and record measured results\n",
+    "1. Add candidates to dataset\n",
+    "1. Retrain on dataset (and poll status w/ wait times)\n",
+    "1. Repeat until max iterations reached"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "sl_results = {}\n",
+    "\n",
+    "best_sl_pred_vals, best_sl_measured_vals = run_design(\n",
```

Comment:
change function name from `run_design` to `run_sequential_learning`

Reply:
done :)